app/workers/deletes.php

```diff
@@ -442,7 +442,7 @@ protected function deleteFunction(Document $document, string $projectId): void
          * Delete builds
          */
         Console::info("Deleting builds for function " . $functionId);
-        $storageBuilds = new Local(APP_STORAGE_BUILDS . '/app-' . $projectId);
+        $storageBuilds = $this->getDevice(APP_STORAGE_BUILDS . '/app-' . $projectId);
```

Comment:
Looks like there's already a [`getBuildsDevice()`](https://github.com/appwrite/appwrite/blob/02a432102df047d2ea96faed65d77a862239d487/src/Appwrite/Resque/Worker.php#L275) method on the Worker. Maybe that can be re-used?

Reply:
done